src/collective/fingerpointing/browser/logview.pt

```diff
@@ -0,0 +1,37 @@
+<html
+    xmlns="http://www.w3.org/1999/xhtml" xml:lang="en"
+    xmlns:tal="http://xml.zope.org/namespaces/tal"
+    xmlns:metal="http://xml.zope.org/namespaces/metal"
+    xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+    lang="en"
+    metal:use-macro="here/prefs_main_template/macros/master"
+    i18n:domain="plone">
+  <body>
+    <div metal:fill-slot="prefs_configlet_main">
+      <a href=""
+         id="setup-link"
+         tal:attributes="href string:$portal_url/plone_control_panel"
+         i18n:translate="">
+        Site Setup
+      </a>
+
+      <h1 class="documentFirstHeading" tal:content="view/label">View Title</h1>
+
+      <div metal:use-macro="context/global_statusmessage/macros/portal_message">
+        Portal status message
+      </div>
+
+      <div id="content-core">
+        <div id="layout-contents">
+          <div style="width: 100%; height: 100%; overflow: scroll;">
+            <pre tal:content="view/logcontents"></pre>
+          </div>
+        </div>
+      </div>
+
+      <p>
+        <a tal:attributes="href string:${context/absolute_url}/@@fingerpointing-log?show_all=1" i18n:translate="link_show_all">Show All</a>
```

Comment:
the basic idea was to implement a quick solution (quick in terms of development time) that fits most use cases.
of course a search feature would be great - but i'd say it's out of scope for this PR

again - i personally think using a log file instead of a database is a bad start for such a feature. (https://github.com/collective/collective.fingerpointing/pull/7#issuecomment-188888432)

as an alternative to the "show all" button we could provide a download button for the meantime.